src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java

```diff
@@ -311,13 +311,13 @@ public void run() {
                 bits,
                 new Object[] { "DPD", "PMP", "DGD", 'M', ItemList.Automation_ChestBuffer_UHV, 'G',
                         ItemList.Field_Generator_ZPM, 'D', OrePrefixes.circuit.get(Materials.UHV), 'P',
-                        OrePrefixes.plateDouble.get(Materials.Europium) });
+                        OrePrefixes.plateTriple.get(Materials.Europium) });
         GTModHandler.addCraftingRecipe(
                 ItemList.Quantum_Chest_IV.get(1L),
                 bits,
                 new Object[] { "DPD", "PMP", "DGD", 'M', ItemList.Automation_ChestBuffer_UEV, 'G',
                         ItemList.Field_Generator_UV, 'D', OrePrefixes.circuit.get(Materials.UEV), 'P',
-                        OrePrefixes.plate.get(Materials.Americium) });
+                        OrePrefixes.plateTriple.get(Materials.Americium) });
```

Comment:
There's an assembler recipe for this one as well somewhere that will need to be updated